cmake/DockerConfig.cmake

```diff
@@ -22,7 +22,6 @@ add_custom_target(
         -u 1000
         -g 1000
         -v ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH}
-        -c IMAGE_TYPE=release
```

Comment:
should we use `-i release` here as other targets use it like that? or should we remove `-i, -u, -g` wherever they align with the defaults in `DockerBuild.sh`?

Reply:
I think you are right, it should be the same parameter list everywhere for the sake of consistency, I added the `-i` switch here as well in c802edc